cli/commands/cloud/cloud_test.go

```diff
@@ -65,23 +69,19 @@ func TestMain(m *testing.M) {
 		panic(err)
 	}
 
-	err = persistence.InitDB(true, "", 0)
+	err = gormX.Init(true, "", 0)
 	if err != nil {
 		panic(err)
 	}
 
-	s = service_orchestrator.NewService()
-
-	sock, server, _, err = service.StartDedicatedAuthServer(":0", service_auth.WithApiKeySaveOnCreate(false))
-	if err != nil {
-		panic(err)
-	}
-	orchestrator.RegisterOrchestratorServer(server, s)
+	orchestratorService = service_orchestrator.NewService(gormX)
+	authService = service_auth.NewService(gormX)
 
-	target, err = s.CreateDefaultTargetCloudService()
+	sock, server, _, err = authService.StartDedicatedAuthServer(":0", service_auth.WithApiKeySaveOnCreate(false))
 	if err != nil {
 		panic(err)
 	}
+	orchestrator.RegisterOrchestratorServer(server, orchestratorService)
```

Comment:
I am not sure if this works. I am pretty sure that you cannot register additional services to a gRPC server once it has been started (which happens in authService.StartDedicatedAuthServer)

Reply:
But this was originally introduced by you :D And the tests don't fail.
I will look at it more closely.

Reply:
I know. The problem is that this seems to creep up on some tests now. Seems this introduces a weird race condition. I have proposed a solution in the #486 branch. Basically adding a function argument to `StartDedicatedAuthServer` that allows the start of additional services.